sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/utils/AvroUtils.java

```diff
@@ -151,9 +151,13 @@ public class AvroUtils {
   }
 
   // Unwrap an AVRO schema into the base type an whether it is nullable.
```

Comment:
Consider replace this inline comment into a brief javadoc for public class

Reply:
thanks, changed this comment